tasks/fix-cat3.yml

```diff
@@ -32,6 +32,34 @@
       - complexity-high
       - passwd
 
+- debug:
+    msg: "{{ dev_shm_mount }}"
+  vars:
+      dev_shm_mount: "{{ ansible_mounts | json_query('[?mount == `/dev/shm`] | [0]') }}"
+  tags:
+    - always
+
+- name: |
+        "LOW | RHEL-07-021022 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the nodev option."
+        "LOW | RHEL-07-021023 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the nosuid option."
+        "LOW | RHEL-07-021024 | PATCH | The Red Hat Enterprise Linux operating system must mount /dev/shm with the noexec option."
+  mount:
+      path: /dev/shm
+      state: mounted
+      src: "{{ dev_shm_mount.device|default('tmpfs') }}"
+      fstype: "{{ dev_shm_mount.fstype|default('tmpfs') }}"
+      opts: "{{ dev_shm_mount.options|default('defaults') }},nodev,nosuid,noexec"
```

Comment:
Yes the mount module is fully idempotent and also handles the remount when its needed.

I was following what we did here: https://github.com/MindPointGroup/RHEL7-CIS/blob/be56adb2f0b4f30b284a8bbc481092d121d1b8f2/tasks/section1.yml#L372

I did go down the path of making this split so that you could do some, all, or none of these settings but is anyone really going to have a need to apply 1 of these options and not the others? If I split these out it would require at a minimum several extra tasks. I can certainly do that but was avoiding the complexity that adds.